ai-services/internal/pkg/cli/helpers/helper.go

```diff
@@ -131,6 +132,52 @@ func FindFreeSpyreCards() ([]string, error) {
 	return free_spyre_dev_id_list, nil
 }
 
+func RunServiceReportContainer(runCmd string, mode string) error {
+	var svc_tool_cmd *exec.Cmd
+	switch mode {
+	case "configure":
+		svc_tool_cmd = exec.Command(
+			"podman",
+			"run",
+			"--privileged",
+			"--rm",
+			"--name", "servicereport",
+			"-v", "/etc/modprobe.d:/etc/modprobe.d",
+			"-v", "/etc/modules-load.d/:/etc/modules-load.d/",
+			"-v", "/etc/udev/rules.d/:/etc/udev/rules.d/",
+			"-v", "/etc/security/limits.d/:/etc/security/limits.d/",
+			vars.ToolImage,
+			"bash", "-c", runCmd,
+		)
+	case "validate":
+		svc_tool_cmd = exec.Command(
+			"podman",
+			"run",
+			"--privileged",
+			"--rm",
+			"--name", "servicereport",
+			"-v", "/etc/group:/etc/group:ro",
+			"-v", "/etc/modprobe.d:/etc/modprobe.d",
+			"-v", "/etc/modules-load.d/:/etc/modules-load.d/",
+			"-v", "/etc/udev/rules.d/:/etc/udev/rules.d/",
+			"-v", "/etc/security/limits.d/:/etc/security/limits.d/",
+			"-v", "/etc/sos:/etc/sos",
```

Comment:
for validate can we have everything readonly?

Reply:
makes sense.